common/tzif.ksy

```diff
@@ -0,0 +1,262 @@
+meta:
+  id: tzif
+  title: The Time Zone Information Format (TZif)
+  xref:
+    mime: application/tzif
+    # mime: "application/tzif-leap" if leap-second records are included
+    rfc: 8536
+  license: CC0-1.0
```

Comment:
Are the comments copued from somewhere? IETF RFCs are copyrighted to IETF.

Reply:
Yes they are BUT IETF grants a license if you attribute:

https://trustee.ietf.org/documents/trust-legal-provisions/tlp-5/

It is in section 3.

Reply:
I probably need to reword all the comments to avoid copyright issues.